config/core/test-configs.yaml

```diff
@@ -2171,6 +2177,24 @@ test_configs:
       - baseline
       - baseline-nfs
 
+  - device_type: juno-uboot
+    test_plans:
+      - baseline
+      - baseline-nfs
+      - kselftest-cpufreq
+      - kselftest-futex
```

Comment:
Failed, looks like something got stuck:
https://lava.sirena.org.uk/scheduler/job/84875

Also it seems to only be an issue with Clang builds, the GCC ones appear to be completing:
https://lava.sirena.org.uk/scheduler/job/84868